src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/Requests/ApprenticeFeedbackTargetUpdateRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure.Api.Requests
```

Comment:
Very finnicky, this plus the Feedback Target for Update are the same object, how much do we care about the repetition ?